web_ui/frontend/app/origin/globus/callback/page.tsx

```diff
@@ -36,26 +36,28 @@ export default function Home() {
 
   const router = useRouter();
 
-  const postCallback = async (search: string): Promise<callbackResponse> => {
-    const url = new URL(
-      '/api/v1.0/origin_ui/globus/auth/callback' + search,
-      window.location.origin
-    );
+  useEffect(() => {
```

Comment:
It looks like this typescript is failing a linter action. I don't know any typescript, so I'll ping Howard for a review when this PR gets a little further along.